news/2019-02-24-project-loved-week-of-february-24th.md

```diff
@@ -0,0 +1,206 @@
+
+A map by captin1, "Lost Time Memory" is a blast from the past considering the map was from 2014, so its patterns maybe a little bit different from what the newer players are used to. You'll be glad to hear though that this mapset doesn't get crazy hard and it's suitable for anyone that plays the game, with the easiest difficulty being just 2 stars and ranging up to mid-5. Give it a shot!
+
+[![](/wiki/shared/news/2019-02-24-project-loved-week-of-february-24th/osu/nexta.jpg)](https://osu.ppy.sh/community/forums/topics/873730)
+
+[lapix - Nexta](https://osu.ppy.sh/beatmapsets/716079#osu) by [DTM9 Nowa](https://osu.ppy.sh/users/5428909), [downpour](https://osu.ppy.sh/users/4543414) and [Nite](https://osu.ppy.sh/users/7401033)  
+*written by [waywern2012](https://osu.ppy.sh/users/5870453)*
+
+Next up we have a map from DTM9 Nowa. This 160 BPM map "Nexta" from Lapix is truly a challenging one. With many difficulties to choose from, they all have a different approach on the map. The theme of the map overall is simple: technical. With a wide variety of spaced jumpstreams, back-and-forth jumps and difficult patterns, especially on downpour's difficulty, it challenges every player that will play this map! Will you take it on? Come prove your technical abilities; it won't be easy!
+
+[![](/wiki/shared/news/2019-02-24-project-loved-week-of-february-24th/osu/chinurareta-shuren-ni-fukeru-houmura.jpg)](https://osu.ppy.sh/community/forums/topics/873729)
+
+[Imperial Circus Dead Decadence - Chinurareta Shuren ni Fukeru Houmura](https://osu.ppy.sh/beatmapsets/508884#osu) by [alkotras42](https://osu.ppy.sh/users/4749499)  
+*written by [waywern2012](https://osu.ppy.sh/users/5870453)*
+
+This is one of the lower-difficulty Imperial Circus Dead Decadence maps that are mapped, but don't let that fool you! This map is a well-made by alkotras42 and still packs in 6.33 stars with quite a bit of streams. The map is simple overall, but it is quite long, making it harder to pass with a full combo without missing a few notes here and there. Coming in at 161 favourites, this map is truly Loved by the community. Give it a go!
+
+[![](/wiki/shared/news/2019-02-24-project-loved-week-of-february-24th/osu/elemental-creation.jpg)](https://osu.ppy.sh/community/forums/topics/873728)
+
+[dj TAKA meets DJ YOSHITAKA - Elemental Creation](https://osu.ppy.sh/beatmapsets/193856#osu) by [DoKito](https://osu.ppy.sh/users/537084)  
+*written by [Bubbleman](https://osu.ppy.sh/users/5182050)*
+
+One of DoKito's infamous maps, Elemental Creation is an older style of map involving many of the winding streams that DoKo is known for, as well as some harsh jump patterns to break up the constant bombardment of flow aim. With two difficulties, one AR9 and one 10, you can pick whether you just want a challenge to mechanics or whether you want to push your reading to the limit, since the song is a pacy 210 BPM. Can you handle the streams? Try your luck!
```

Comment:
```suggestion
One of DoKito's infamous maps, Elemental Creation is an older style of map involving many of the winding streams that DoKo is known for, as well as some harsh jump patterns to break up the constant bombardment of flow aim. With two difficulties, one AR9 and one 10, you can pick whether you just want a challenge to mechanics or whether you want to push your reading to the limit, since the song is a pacy 212 BPM. Can you handle the streams? Try your luck!
```